cranial/messaging/s3.py

```diff
@@ -0,0 +1,123 @@
+import json
+import os
+from typing import Dict, IO  # noqa
+
+from cranial.messaging import base
+from cranial.common import logger
+from cranial.connectors import FileConnector
+
+log = logger.get()
+
+
```

Comment:
Why so much duplication in this file? I'm not sure we need it at all. At most, it could just be `Notifier = cranial.messaging.file.Notifier`, right?

Reply:
The thing is, looking through the code, cranial.messaging.file.Notifier has a lot of safeties and checks based off the os package. These checks fail for s3. I didn't want to take them out because I thought they might be useful.